transaction-service/src/main/java/pl/adamboguszewski/transaction/service/application/TransactionPayment.java

```diff
@@ -10,21 +10,21 @@
 
     Long multiplier;
 
-    TransactionCurrency transactionCurrency;
+    Currency currency;
 
     TransactionPaymentType transactionPaymentType;
 
-    public TransactionPayment(Long amountPaid, Long multiplier, TransactionCurrency transactionCurrency,
+    public TransactionPayment(Long amountPaid, Long multiplier, Currency currency,
```

Comment:
Shouldn't this constructor be private? (Anyway, it would be my error)

Reply:
yes